src/main/java/org/broadinstitute/consent/http/service/DatabaseMatchingServiceAPI.java

```diff
@@ -39,12 +39,12 @@
     private final Logger logger = LoggerFactory.getLogger(this.getClass());
 
     public static void initInstance(Client client, DataAccessRequestDAO dataAccessRequestDAO,
-        ServicesConfiguration config, DatasetService datasetService, UseRestrictionConverter useRestrictionConverter) {
-        MatchAPIHolder.setInstance(new DatabaseMatchingServiceAPI(client, dataAccessRequestDAO, config, AbstractConsentAPI.getInstance(), datasetService, useRestrictionConverter));
+        ServicesConfiguration config, ConsentService consentService, DatasetService datasetService, UseRestrictionConverter useRestrictionConverter) {
+        MatchAPIHolder.setInstance(new DatabaseMatchingServiceAPI(client, dataAccessRequestDAO, config, consentService, datasetService, useRestrictionConverter));
     }
 
-    DatabaseMatchingServiceAPI(Client client, DataAccessRequestDAO dataAccessRequestDAO, ServicesConfiguration config, ConsentAPI consentAPI, DatasetService datasetService, UseRestrictionConverter useRestrictionConverter){
-        this.consentAPI = consentAPI;
+    DatabaseMatchingServiceAPI(Client client, DataAccessRequestDAO dataAccessRequestDAO, ServicesConfiguration config, ConsentService consentService, DatasetService datasetService, UseRestrictionConverter useRestrictionConverter){
+        this.consentService = consentService;
```

Comment:
This is a case where we want to try and minimize pulling one service into another. I see that there are only two usages here that we care about, maybe we can instead pull in the consent DAO and make those db calls directly.